core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -4415,6 +4481,24 @@ object Stream extends StreamLowPriority {
     def toVector: Either[Throwable, Vector[O]] = to(Vector)
   }
 
+  /** Provides syntax for `IO` streams. */
+  implicit final class IOOps[A](private val self: Stream[IO, A]) extends AnyVal {
+
+    /** Creates a [[Publisher]] from this [[Stream]].
+      *
+      * The stream is only ran when elements are requested.
+      *
+      * @note This [[Publisher]] can be reused for multiple [[Subscribers]],
+      *       each [[Subscription]] will re-run the [[Stream]] from the beginning.
+      *
+      * @see [[toPublisher]] for a safe version that returns a [[Stream]].
+      */
+    def unsafeToPublisher()(implicit
+        runtime: IORuntime
+    ): Publisher[A] =
+      interop.flow.unsafeToPublisher(self)
+  }
```

Comment:
This is the one I'm most interested in. It is the `Stream` analog of `io.unsafeToFuture()`.

The motivation to facilitate easy interop with other libraries e.g. https://github.com/raquo/Airstream/pull/114:
```scala
EventStream.fromFlowPublisher(fs2Stream.unsafeToPublisher())
```